marcin-tupaj/game-points
# [TICKET-2137] Add Kahoot Points game board

### Kahoot! Points - Game board

Dark version:
- <img width="1303" alt="Screenshot 2023-05-21 at 21 59 03" src="https://github.com/marcin-tupaj/game-points/assets/60094543/bb7186b4-3314-4106-892f-794051793764">


Light version:
- <img width="1306" alt="Screenshot 2023-05-21 at 22 01 01" src="https://github.com/marcin-tupaj/game-points/assets/60094543/ba1df1b7-ea57-4d2f-8e7e-29687914d086">


Mobile:
- ![localhost_5173_(Pixel 5) (1)](https://github.com/marcin-tupaj/game-points/assets/60094543/daae2f02-c747-49ac-bd41-62d197612c6d)



Walkthrough:

https://github.com/marcin-tupaj/game-points/assets/60094543/b26ddd54-6a50-4995-b4f1-72c4eba8a08f



Testing notes:
- check both the light and dark versions
- check mobile, tablet, and desktop versions
- check that multiple bonuses are applied, use A >= 6 times (bonus for the quantity of 3) or B >=4 times (bonus for the quantity of 2)
- check the _new game_ option
- check keyboard support

RELEASES Add Kahoot Points game board #ui closes [[TICKET-2137]](https://jira.com)
